conditional/util/context_processors.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=bare-except
+
+from conditional import app
+from conditional.models.models import FreshmanAccount
+from conditional.util.cache import service_cache
+from conditional.util.ldap import ldap_get_member, ldap_is_current_student
+
+
+def get_member_name(uid):
+    try:
+        return get_csh_name(uid)
+    except:
+        return get_freshman_name(uid)
+
+
+@service_cache(maxsize=256)
+def get_csh_name(username):
```

Comment:
What's the difference between `uid` and `username` throughout this PR?

If they're both referring to CSH uids, I'd prefer to use `uid` throughout.

Reply:
So the whole project uses them interchangeably everywhere. I agree, but that would require touching so many files

Reply:
Could we be consistent moving forward? You’ve used them interchangchably in the same files for similar functions, even for stuff you’re adding now.